### PR TITLE
Add loggging for SSR issues, fix empty collection case.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: src/route/routesSSRData.js [hunk 1]
```diff
@@ -214,7 +218,14 @@ export const playlistCollectionPage = (store, match) => {
...
     .then(() => {
       const [c] = mdbGetCollectionByIdSelector(store.getState(), [cID]);
+      if (!cuId && !c?.cuIDs) {
+        // This can happen when collection is empty (unit is not secure or published).
+        console.error(`Failed fetching unit for ${cuId} for ${cID}`);
+        return Promise.reject();
+      }
+
       store.dispatch(mdbActions.fetchUnit(cuId || c?.cuIDs[0]));
+      return Promise.resolve(null);
```
Comment (davgur) [inline src/route/routesSSRData.js:228]: you dont need return Promise.resolve .then wrap to promise automate what you return

**Review by davgur — APPROVED** [review]

File: src/route/routesSSRData.js (hunk 1 above)
Comment (bbfsdev) [reply src/route/routesSSRData.js:228]: I have to as I have return previously so Lint fails on arrow function should have a return value

Comment (davgur) [reply src/route/routesSSRData.js:228]: you can return null, but as I understand that you want error, so its good

Comment (bbfsdev) [reply src/route/routesSSRData.js:228]: Yes.

**Pull request merged**